Portfolio1.html

```diff
@@ -9,7 +9,8 @@
 
 <body>
 <!--Linked Logo and Internal Link -->
-
+<a target="_blank" href="https://www.history.com/topics/ancient-history/tutankhamen"> <img src="pexels-pixabay-33571.jpg" alt="Gold plated crest from the sarcophagi of Tutankharmun"></a>
```

Comment:
<img> tag to go on a separate line. the closing </a> tag to go on the line after